### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sakshamtaneja21**
Have done some changes to fig grammatical errors 

## Checklist

- [ ] I ran `npm test` locally and it passed without errors.

- [ ] I only edited the `_data/pixels.json` file.

- [ ] I entered the `username` in the `pixels.json` that I'm also using to create this pull request.
- [ ] I acknowledge that all my contributions will be made under the project's [license](../../LICENSE.md).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,24 +2,24 @@
 
 [![All Contributors](https://img.shields.io/badge/all_contributors-12-orange.svg?style=flat-square)](#contributors) [![A Twilio Labs Project](https://img.shields.io/static/v1?label=&message=Twilio-Labs&color=F22F46&labelColor=0D122B&logo=twilio&style=flat-square)](https://www.twilio.com/labs) [![Netlify Status](https://api.netlify.com/api/v1/badges/611ac0f9-4ae9-48a2-9769-26c32cb5f9e8/deploy-status)](https://app.netlify.com/sites/pixel-project-dev/deploys)
 
-This is an art project created with the intention to teach you how to create a pull request.
+This is an art project created to teach you how to create a pull request.
```
Comment (Kiy4h) [inline README.md]: I don't think this change is required.

File: README.md [hunk 2]
```diff
@@ -2,24 +2,24 @@
...
 
 Visit [open-pixel-art.com](https://open-pixel-art.com) to see the full artwork!
 
-Anyone who wishes to learn how to create a pull request for a project on GitHub can use this project to contribute a pixel to the canvas. You will only be able to ever create one pixel but you can pix whatever color you prefer as long as it is a valid HEX code. For example: `#F22F46` the brand color of [Twilio](https://www.twilio.com)
+Anyone who wishes to learn how to create a pull request for a project on GitHub can use this project to contribute a pixel to the canvas. You will only be able to ever create one pixel but you can pix whatever colour you prefer as long as it is a valid HEX code. For example: `#F22F46` the brand color of [Twilio](https://www.twilio.com)
```
Comment (Kiy4h) [inline README.md]: I don't think this should be changed either

File: README.md [hunk 3]
```diff
@@ -2,24 +2,24 @@
...
 > 🕹 [Download TwilioQuest](https://www.twilio.com/quest/download)
 
 The entire project is automated and is largely maintained by a set of bots that will verify any pixel contributions. However, if you'd like to know more about the project or submit other contributions to the project that are not a pixel, feel free to create a [GitHub issue](https://github.com/twilio-labs/open-pixel-art/issues) inside the [Open Pixel Art project](https://github.com/twilio-labs/open-pixel-art).
 
 ## Contributing
 
-In order to contribute a pixel to the canvas, you'll have to create a [pull request](https://opensource.guide/how-to-contribute/#opening-a-pull-request) to the [Open Pixel Art project on GitHub](https://github.com/twilio-labs/open-pixel-art).
+To contribute a pixel to the canvas, you'll have to create a [pull request](https://opensource.guide/how-to-contribute/#opening-a-pull-request) to the [Open Pixel Art project on GitHub](https://github.com/twilio-labs/open-pixel-art).
```
Comment (Kiy4h) [inline README.md]: There's no need to simplify this, it should be fine without your changes :)

File: README.md (hunk 2 above)
Comment (dkundel) [reply README.md]: +1 on this one. Let's keep this to American English

File: README.md [hunk 4]
```diff
@@ -2,24 +2,24 @@
...
+Anyone who wishes to learn how to create a pull request for a project on GitHub can use this project to contribute a pixel to the canvas. You will only be able to ever create one pixel but you can pix whatever colour you prefer as long as it is a valid HEX code. For example: `#F22F46` the brand color of [Twilio](https://www.twilio.com)
 
-> If you are new to open-source, GitHub or git in general, fear not, we created a tutorial in [TwilioQuest](https://www.twilio.com/quest) for you. It will teach you step by step on how you can get started and will guide you on your quest of creating your pull request.
+> If you are new to open-source, GitHub or git in general, fear not, we created a tutorial in [TwilioQuest](https://www.twilio.com/quest) for you. It will teach you to step by step on how you can get started and will guide you on your quest of creating your pull request.
```
Comment (dkundel) [inline README.md]: This seems wrong. Could you please revert this?

File: README.md [hunk 5]
```diff
@@ -60,7 +60,7 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
   </tr>
 </table>
 
-<!-- ALL-CONTRIBUTORS-LIST:END -->
+<!-- ALL-CONTRIBUTORS-LIST: END -->
```
Comment (dkundel) [inline README.md]: This breaks All Contributors. Please revert this

File: README.md (hunk 3 above)
Comment (dkundel) [reply README.md]: Not needed but I'm fine with this one

**Comment by dkundel** [comment]
Left some more comments :) Thank you for your review @Kiy4h! 🎉

File: README.md (hunk 2 above)
Comment (sakshamtaneja21) [reply README.md]: ok ill change it

**Comment by sakshamtaneja21** [comment]
how can I get an invite of your GitHub organization as a member

**Pull request closed without merging**